packages/frontend/amp/components/Ad.tsx

```diff
@@ -37,6 +37,36 @@ const adStyle = css`
     }
 `;
 
+const adClass = css`
+    display: none;
+`;
+
+const usAdRegionClass = css`
+    .amp-geo-group-us & {
+        display: block;
+    }
+`;
+
+const auAdRegionClass = css`
+    .amp-geo-group-au & {
+        display: block;
+    }
+`;
+
+const rowAdRegionClass = css`
+    .amp-geo-group-eea & {
+        display: block;
+    }
+`;
+
+const regionClasses = {
```

Comment:
Similar to below:
```suggestion
const adRegionClasses = {
```

Reply:
Agree. Changed in new commit. Pushing it next.